docs/TRANSLATIONS.md

```diff
@@ -1,116 +1,47 @@
 # Translations
 
-## Help us to review/proofread translations
+## For Translators
 
-You can join our [GitHub translation teams](https://github.com/orgs/organicmaps/teams/translations/teams),
-so any contributor can tag all teams (or a specific language team) to get help with the review.
+Translations are maintained using Weblate:
 
-Please respond in the relevant [GitHub discussion](https://github.com/orgs/organicmaps/discussions/8538), or let us know at hello@organicmaps.app
+https://hosted.weblate.org/projects/organicmaps/
 
-## Contribute translations directly
```

Comment:
Need to add a "How to become a translator" / coordinator

Reply:
I added basic instructions. Let's extend later.